testing-guidance/workflow-tests/animation/animation-editor-workflows.md

```diff
@@ -0,0 +1,120 @@
+# Animation Editor Workflow Tests
+
+Testing in this area should focus on the general usage workflows of the Animation Editor.
+
+
+## General Docs
+* [O3DE Animation Editor Concepts and Terms](https://www.o3de.org/docs/user-guide/visualization/animation/character-editor/concepts-and-terms/)
+* [O3DE Animation Editor User Interface](https://www.o3de.org/docs/user-guide/visualization/animation/animation-editor/user-interface/)
+* [O3DE Getting Started With Animation Editor](https://www.o3de.org/docs/user-guide/visualization/animation/animation-editor/quick-start/)
+
+## Common Issues to Watch For
+
+Test guidance will sometimes note specific issues to watch for. The common issues below should be watched for through all testing, even if unrelated to the current workflow being tested.
+- Asset processor errors when saving animation settings
+- Warnings or Errors that appear in the Editor Console Log while setting up or running scenes.
+- Errors appearing in the Animation Editor's Log Window view.
+- Broken Animation behaviors.
+
+
+## Workflow Variants
+### Source Nodes
+* Bind Pose
+* State Machine
+* Motion
+* Entry
+* Reference
+* Blend Tree
+* Hub
+
+## Workflows
+
+### Area: Animation Editor Behaviors
+
+**Description:** Testing in this area will focus on basic Animation Editor behaviors such as docking, editor layout and management, Actor importing, and animation rendering.
+
+**Project Requirements:**
+
+Any project that has the following Gems enabled: 
+* EMotion FX Animation
+* Scripted Entity Tweener
+* Primitive Assets
```

Comment:
suggestion: It might be useful to include paths to where the Gems are located to make finding them easier

Reply:
Added